.circleci/config.yml

```diff
@@ -4,18 +4,23 @@ jobs:
     build:
         resource_class: medium+
         docker:
-            - image: circleci/node:9-browsers
+            - image: nikolaik/python-nodejs:python3.7-nodejs8
         environment:
             CONTRACTS_COMMIT_HASH: '9ed05f5'
         working_directory: ~/repo
         steps:
             - checkout
             - run: echo 'export PATH=$HOME/CIRCLE_PROJECT_REPONAME/node_modules/.bin:$PATH' >> $BASH_ENV
-            - run:
-                  # HACK(albrow): Without this, yarn commands will sometimes
-                  # fail with a "permission denied" error.
-                  name: Set npm path
-                  command: npm set prefix=/home/circleci/npm && echo 'export PATH=$HOME/circleci/npm/bin:$PATH' >> /home/circleci/.bashrc
+            # HACK(feuGeneA): commented out this hack as we're changing
+            # from a circleci-maintained container to a different
+            # container, and this hack may not apply anymore, as
+            # suggested by the non-existance of `/home/circleci/.bashrc`
+            # when running the command below.
+            # - run:
+            #       # HACK(albrow): Without this, yarn commands will sometimes
+            #       # fail with a "permission denied" error.
+            #       name: Set npm path
+            #       command: npm set prefix=/home/circleci/npm && echo 'export PATH=$HOME/circleci/npm/bin:$PATH' >> /home/circleci/.bashrc
```

Comment:
And? Are we able to remove it?

Reply:
I don't think we can say for sure just yet.  It was an intermittent problem.  But I have _not_ experienced that intermittent problem since disabling it, and I've been through probably 6 or 7 builds now.